frontend/src/helpers/video/get-format-duration-string.helper.ts

```diff
@@ -0,0 +1,20 @@
+import dayjs from 'dayjs';
+import * as dayjsDuration from 'dayjs/plugin/duration';
+
+dayjs.extend(dayjsDuration.default);
+
+const getFormatDurationString = (durationInSeconds: number): string => {
+  const d = dayjs.duration(durationInSeconds * 1000);
+  if (d.hours() > 9) {
+    return d.format('HH:mm:ss');
+  }
+  if (d.hours()) {
+    return d.format('H:mm:ss');
+  }
+  if (d.minutes() > 9) {
+    return d.format('mm:ss');
+  }
+  return d.format('m:ss');
+};
```

Comment:
```suggestion
  if (d.hours()) {
    return d.format('H:mm:ss');
  }
  return d.format('m:ss');
};
```
too many redundant checks.

Reply:
redirected the issue to @Barracuda713